docs/internships/outreachy/getting-started.md

```diff
@@ -0,0 +1,46 @@
+---
+id: getting-starteds
+title: Our repositories
+sidebar_label: Getting Started
+---
+
+As part of the application process, all applicants must make at least one contribution to be accepted as an intern for this project. Applicants can contribute to our project through our github repositories.
+1. [Talawa mobile github](https://github.com/PalisadoesFoundation/talawa)
```

Comment:
Add a space above. Sometimes markdown requires that for bullets to be recognised.